src/index.html

```diff
@@ -4,9 +4,383 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo header__logo" href="index.html">
+        <img src="./images/logo.png" alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              class="nav__link" 
+              href="#laptops&computers" 
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div class="catalog__filling">
+          <div class="card catalog__card" data-qa="card">
+            <img 
+              class="card__image" 
+              src="./images/imac.jpeg" 
+              alt="iMac image"
+            >
+      
+            <div class="card__info">
+              <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      
+              <p class="card__code">Product code: 195434</p>
+            </div>
+          
+            <div class="card__review-line">
+              <div class="stars stars--4">
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+              </div>
+      
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+      
+            <div class="card__price">
+              <p class="card__price-line">Price:</p>
+              
+              <p class="card__price-value">$2,199</p>
+            </div>
+      
+            <a 
+              class="card__button" 
+              href="/" 
+              data-qa="card-hover"
+              >
```

Comment:
Please check your code and refactor according comments
```suggestion
            >
```